dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/transport/AbstractClient.java

```diff
@@ -344,14 +363,14 @@ public void reconnect() throws RemotingException {
     @Override
     public void close() {
         try {
-            if (executor != null) {
-                ExecutorUtil.shutdownNow(executor, 100);
-            }
+            super.close();
```

Comment:
`reconnectExecutorService.shutdownNow();`
How about shutdown the scheduler here.

Reply:
1. `reconnectExecutorService` is a shared instance, this can not be closed, used to handle other client reconnection
2. If it is shutdown, the reconnection caused by network jitter will be invalid.